Sources/SwiftSyntaxMacroExpansion/MacroSystem.swift

```diff
@@ -409,10 +409,13 @@ private func expandBodyMacro(
     return nil
   }
 
-  // Wrap the body in braces.
-  let beforeBody = decl.body == nil ? " " : "";
-  let indentedSource = beforeBody + "{\n" + expanded.indented(by: decl.indentationOfFirstLine + indentationWidth) + "\n}\n"
-  return "\(raw: indentedSource)" as CodeBlockSyntax
+  // `expandAttachedMacro` adds the `{` and `}` to wrap the accessor block and
+  // then indents it.
+  // Remove any indentaiton from the first line using `drop(while:)` and then
+  // prepend a space to separate it from the variable declaration
```

Comment:
I assume this was copy pasted? Doesn't really apply to the function body, although the behavior is probably still what we want.

Reply:
Ah, Alex already commented (I swear I refreshed but apparently not).